soap/src/java/com/zimbra/soap/mail/type/EditheaderTest.java

```diff
@@ -224,11 +225,19 @@ public String toString() {
 
     public void validateEditheaderTest() throws ServiceException {
         if (StringUtil.isNullOrEmpty(headerName)) {
-            throw ServiceException.PARSE_ERROR("Invalid EditheaderTest: Missing headerName", null);
```

Comment:
most of the newly added validations exist in EditeHeaderExtension.commonValidation() method. should try to remove the duplicate code, if not atleast have the exception message same.